styles.css

```diff
@@ -12,12 +12,6 @@ body {
   letter-spacing: 1px;
 }
 
-button {
-  cursor: pointer;
-  background: none;
-  border: none;
-}
-
 .container {
   width: 100%;
   margin: 0 auto;
```

Comment:
- [ ] (Optional) Kindly make the header for the desktop version look like that of the Figma template that you chose. Please remove the `hamburger` button and add three links to the right. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/160239516-4ff90c0d-dec0-456a-928a-6fb82b2f29c9.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/160239571-44cc5ab2-53de-42c3-a9dd-d46db1fa4535.png)